_foundations/linux.md

```diff
@@ -12,65 +12,65 @@ sidebar:
   - image: "assets/images/linux.svg"
     image_alt: "Linux Foundation logo"
     text: >
-      Tax Status: 501C6  
+      Tax Status: The Linux Foundation uses multiple entity types, including 501C3, 501C4, and 501C6. The Linux Foundation is a 501C6.  
```

Comment:
Note that currently, the nonprofit/"Tax Status" field is for the parent foundation itself, not for the (collaborative) projects hosted at the foundation. So for now I'll be marking this as just 501C6 for the LF as a whole.

Reply:
Gotcha.